Documentation/Decode-Root-Keys.md

```diff
@@ -0,0 +1,26 @@
+The easiest way to decode models from json is to use the global `decode`
+function. You can pass the `AnyObject` variable returned from
+`NSJSONSerialization` directly to `decode` and get the model back. However,
+many times the model json is embedded within a root key:
+
+```
+{
+  "user": {
+    "id": 1881372911,
+    "name": "George Sr"
+  }
+}
+```
+
+In this case, you can't use the global `decode` function. The easiest way
```

Comment:
The easiest way what?

You've already said "the easiest way" at the beginning of this document, so what about:

> In this case, you can't use the global `decode` function [edit: is there a specific reason why?]. To get around this, first parse the `AnyObject` into `JSON`, then use the `<|` operator to ... [what are you doing here?]